src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/CompareSymbolsCorrectlyAnalyzer.cs

```diff
@@ -16,6 +18,10 @@ public class CompareSymbolsCorrectlyAnalyzer : DiagnosticAnalyzer
         private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(CodeAnalysisDiagnosticsResources.CompareSymbolsCorrectlyDescription), CodeAnalysisDiagnosticsResources.ResourceManager, typeof(CodeAnalysisDiagnosticsResources));
 
         private static readonly string s_symbolTypeFullName = typeof(ISymbol).FullName;
+        private static readonly string[] s_comparerTypeFullName = new[] {
+            typeof(IEqualityComparer<>).FullName,
```

Comment:
Consider adding these to WellKnownTypes instead of defining here.

Reply:
It's already defined there:

https://github.com/dotnet/roslyn-analyzers/blob/6f404368ab301d1ace6d811ef23fe7cdee8afd5d/src/Utilities/Compiler/WellKnownTypes.cs#L361-L364

Reply:
It looks like `IComparer<T>` would need to be added though.

Reply:
Sure, I'll add.